### ISSUE TITLE
Improve the UI for Console tables

### ISSUE THREAD

**Issue by skyrpex** [winglang/wing#4581]
Currently, the UI for tables is not very good:

- Tries to show all columns at the same time
- The content is cut if it doesn't fit the column
- Can't resize columns

<img width="726" alt="image" src="https://github.com/winglang/wing/assets/1077520/43a1f4c5-3f59-41c5-9723-5f9111ae3f57">

I'm a big fan of the [SLDS](https://www.lightningdesignsystem.com/), specially their [tables](https://www.lightningdesignsystem.com/components/data-tables/).

<img width="815" alt="image" src="https://github.com/winglang/wing/assets/1077520/b55a9aa3-1650-4bc2-a271-85c566025ddf">

In short, their tables are:

- Resizable and scrollable
- User can choose to either trim the content of a column, or break it into multiple lines (can't be seen in the component blueprint but that's how it works in Salesforce)

**Comment by skyrpex** [comment]
Also, https://tanstack.com/table/v8 is a very nice choice. It's a headless table.

**Comment by skyrpex** [comment]
Interesting article about data tables: https://bootcamp.uxdesign.cc/data-table-design-patterns-4e38188a0981.
